ampcrowd/basecrowd/urls.py

```diff
@@ -8,4 +8,6 @@
     url(r'^(\w+)/responses/$', views.post_response, name='post_response'),
     url(r'^(\w+)/tasks/$', views.create_task_group, name='create_tasks'),
     url(r'^(\w+)/purge_tasks/$', views.purge_tasks, name='purge_tasks'),
+    url(r'^(\w+)/summary/task_groups', views.get_task_groups, name='get_task_groups'),
+    url(r'^(\w+)/summary/tasks', views.get_tasks, name="get_tasks")
 )
```

Comment:
`summary` is not, I just didn't have much time to spend on making sure there were no url collisions (got it ready 30 minutes before presentation). I can clean this up.